### PR TITLE
Fixes inconsistencies in Social Login setup guides

### PR DESCRIPTION
**Pull request by bellatumaneng-okta**
## Description:
- Fixes inconsistencies in Social Login setup guide + specify where to configure an Identity Provider in the Developer Console and Classic UI.

### Resolves:

* [OKTA-232350](https://oktainc.atlassian.net/browse/OKTA-232350)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/sign-in-with-facebook/configure-idp/index.md [hunk 1]
```diff
@@ -4,13 +4,13 @@ title: Configure Facebook as an Identity Provider in Okta
 
 2.1. Sign in to your Okta org.
 
-2.2. On the main page, hover your cursor over **Users** until the menu opens, then click on **Social & Identity Providers**.
+2.2. If you use the Developer Console, hover your cursor over **Users** until the menu opens, then click on **Social & Identity Providers**. If you use the Classic UI, hover over **Security** then click on **Identity Providers**.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-in-with-facebook/configure-idp/index.md]: If you use the Developer Console, hover your cursor over **Users** until the menu opens, and then click **Social & Identity Providers**. If you use the Classic UI, hover over **Security**, and then click **Identity Providers**.

File: packages/@okta/vuepress-site/docs/guides/sign-in-with-facebook/configure-idp/index.md [hunk 2]
```diff
@@ -4,13 +4,13 @@ title: Configure Facebook as an Identity Provider in Okta
...
 
 2.3. On the Identity Providers page, click on **Add Identity Provider** > **Add Facebook**.
 
 * **Name:** We suggest using the name you would expect to see on a button, something like "Log in to Facebook".
-* **Client Id:** Paste in the App ID that you got from Facebook in step 1.5 above.
-* **Client Secret:** Paste in the App Secret that you got from Facebook in step 1.5 above.
+* **Client Id:** Paste in the App ID that you got from Facebook in step 1.5.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-in-with-facebook/configure-idp/index.md]: **Client Id:** Paste the App ID that you got from Facebook in step 1.5.

File: packages/@okta/vuepress-site/docs/guides/sign-in-with-facebook/configure-idp/index.md [hunk 3]
```diff
@@ -4,13 +4,13 @@ title: Configure Facebook as an Identity Provider in Okta
...
+* **Client Secret:** Paste in the App Secret that you got from Facebook in step 1.5.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-in-with-facebook/configure-idp/index.md]: **Client Secret:** Paste the App Secret that you got from Facebook in step 1.5.

File: packages/@okta/vuepress-site/docs/guides/sign-in-with-google/configure-idp/index.md [hunk 4]
```diff
@@ -6,13 +6,13 @@ title: Configure Google as an Identity Provider in Okta
 
 2.2. On the main page, click on the **Admin** button in the upper right.
 
-2.3. Hover your cursor over **Users** until the menu opens, then click on **Social & Identity Providers**.
+2.3. If you use the Developer Console, hover your cursor over **Users** until the menu opens, then click on **Social & Identity Providers**. If you use the Classic UI, hover over **Security** then click on **Identity Providers**.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-in-with-google/configure-idp/index.md]: If you use the Developer Console, hover your cursor over **Users** until the menu opens, and then click **Social & Identity Providers**. If you use the Classic UI, hover over **Security**, and then click **Identity Providers**.

File: packages/@okta/vuepress-site/docs/guides/sign-in-with-google/configure-idp/index.md [hunk 5]
```diff
@@ -6,13 +6,13 @@ title: Configure Google as an Identity Provider in Okta
...
 
 2.4. On the Identity Providers page, click on **Add Identity Provider** > **Add Google**.
 
 * **Name:** We suggest using the name you would expect to see on a button, something like "Log in to Google".
-* **Client Id:** Paste in the App ID that you got from Google in step 1.3 above.
-* **Client Secret:** Paste in the App Secret that you got from Google in step 1.3 above.
+* **Client Id:** Paste in the OAuth client ID that you got from Google in step 1.3.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-in-with-google/configure-idp/index.md]: **Client Id:** Paste the OAuth client ID that you got from Google in step 1.3.

File: packages/@okta/vuepress-site/docs/guides/sign-in-with-google/configure-idp/index.md [hunk 6]
```diff
@@ -6,13 +6,13 @@ title: Configure Google as an Identity Provider in Okta
...
+* **Client Secret:** Paste in the OAuth client secret that you got from Google in step 1.3.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-in-with-google/configure-idp/index.md]: **Client Secret:** Paste the OAuth client secret that you got from Google in step 1.3.

File: packages/@okta/vuepress-site/docs/guides/sign-in-with-linkedin/configure-idp/index.md [hunk 7]
```diff
@@ -4,13 +4,13 @@ title: Configure LinkedIn as an Identity Provider in Okta
 
 2.1. Sign in to your Okta org.
 
-2.2. On the main page, hover your cursor over **Users** until the menu opens, then click on **Social & Identity Providers**.
+2.2. If you use the Developer Console, hover your cursor over **Users** until the menu opens, then click on **Social & Identity Providers**. If you use the Classic UI, hover over **Security** then click on **Identity Providers**.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-in-with-linkedin/configure-idp/index.md]: If you use the Developer Console, hover your cursor over **Users** until the menu opens, and then click **Social & Identity Providers**. If you use the Classic UI, hover over **Security**, and then click **Identity Providers**.

File: packages/@okta/vuepress-site/docs/guides/sign-in-with-linkedin/configure-idp/index.md [hunk 8]
```diff
@@ -4,13 +4,13 @@ title: Configure LinkedIn as an Identity Provider in Okta
...
 
 2.3. On the Identity Providers page, click on **Add Identity Provider** > **Add LinkedIn**
 
 * **Name:** We suggest using the name you would expect to see on a button, something like "Log in to LinkedIn".
-* **Client Id:** Paste in the App ID that you got from LinkedIn in step 1.3 above.
-* **Client Secret:** Paste in the App Secret that you got from LinkedIn in step 1.3 above.
+* **Client Id:** Paste in the OAuth client ID that you got from LinkedIn in step 1.3.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-in-with-linkedin/configure-idp/index.md]: **Client Id:** Paste the OAuth client ID that you got from LinkedIn in step 1.3.

File: packages/@okta/vuepress-site/docs/guides/sign-in-with-linkedin/configure-idp/index.md [hunk 9]
```diff
@@ -4,13 +4,13 @@ title: Configure LinkedIn as an Identity Provider in Okta
...
+* **Client Secret:** Paste in the OAuth client secret that you got from LinkedIn in step 1.3.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-in-with-linkedin/configure-idp/index.md]: **Client Secret:** Paste the OAuth client secret that you got from LinkedIn in step 1.3.

File: packages/@okta/vuepress-site/docs/guides/sign-in-with-microsoft/configure-idp/index.md [hunk 10]
```diff
@@ -1,18 +1,18 @@
...
-title: Configure Facebook as an Identity Provider in Okta
+title: Configure Microsoft as an Identity Provider in Okta
 ---
 
 2.1. Sign in to your Okta org.
 
 2.2. On the main page, click on the **Admin** button in the upper right.
 
-2.3. Hover your cursor over **Users** until the menu opens, then click on **Social & Identity Providers**.
+2.3. If you use the Developer Console, hover your cursor over **Users** until the menu opens, then click on **Social & Identity Providers**. If you use the Classic UI, hover over **Security** then click on **Identity Providers**.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-in-with-microsoft/configure-idp/index.md]: If you use the Developer Console, hover your cursor over **Users** until the menu opens, and then click **Social & Identity Providers**. If you use the Classic UI, hover over **Security**, and then click **Identity Providers**.

File: packages/@okta/vuepress-site/docs/guides/sign-in-with-microsoft/configure-idp/index.md [hunk 11]
```diff
@@ -1,18 +1,18 @@
...
 
 2.4. On the Identity Providers page, click on **Add Identity Provider** > **Add Microsoft**.
 
 * **Name:** We suggest using the name you would expect to see on a button, something like "Log in to Microsoft".
-* **Client Id:** Paste in the App ID that you got from Microsoft in step 1.3 above.
-* **Client Secret:** Paste in the App Secret that you got from Microsoft in step 1.3 above.
+* **Client Id:** Paste in the App ID that you got from Microsoft in step 1.3.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-in-with-microsoft/configure-idp/index.md]: **Client Id:** Paste the App ID that you got from Microsoft in step 1.3.

File: packages/@okta/vuepress-site/docs/guides/sign-in-with-microsoft/configure-idp/index.md [hunk 12]
```diff
@@ -1,18 +1,18 @@
...
+* **Client Secret:** Paste in the App Secret that you got from Microsoft in step 1.3.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-in-with-microsoft/configure-idp/index.md]: **Client Secret:** Paste the App Secret that you got from Microsoft in step 1.3.

**Comment by bellatumaneng-okta** [comment]
Screen Shot 2019-06-13 at 3 40 26 PM
Screen Shot 2019-06-13 at 3 40 12 PM
Screen Shot 2019-06-13 at 3 39 57 PM
Screen Shot 2019-06-13 at 3 39 22 PM

File: packages/@okta/vuepress-site/docs/guides/sign-in-with-microsoft/configure-idp/index.md [hunk 13]
```diff
@@ -1,18 +1,18 @@
...
-2.3. Hover your cursor over **Users** until the menu opens, then click on **Social & Identity Providers**.
+2.3. If you use the Developer Console, hover your cursor over **Users** until the menu opens, and then click **Social & Identity Providers**. If you use the Classic UI, hover over **Security**, and then click **Identity Providers**.
 
 2.4. On the Identity Providers page, click on **Add Identity Provider** > **Add Microsoft**.
 
 * **Name:** We suggest using the name you would expect to see on a button, something like "Log in to Microsoft".
-* **Client Id:** Paste in the App ID that you got from Microsoft in step 1.3 above.
-* **Client Secret:** Paste in the App Secret that you got from Microsoft in step 1.3 above.
+* **Client Id:** Paste the App ID that you got from Microsoft in step 1.3.
+* **Client Secret:** Paste the App Secret that you got from Microsoft in step 1.3.
```
Comment (larsjohansen-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-in-with-microsoft/configure-idp/index.md]: Is it called App Secret in the Microsoft terms?

File: packages/@okta/vuepress-site/docs/guides/sign-in-with-facebook/configure-idp/index.md [hunk 14]
```diff
@@ -4,13 +4,13 @@ title: Configure Facebook as an Identity Provider in Okta
 
 2.1. Sign in to your Okta org.
 
-2.2. On the main page, hover your cursor over **Users** until the menu opens, then click on **Social & Identity Providers**.
+2.2. If you use the Developer Console, hover your cursor over **Users** until the menu opens, and then click **Social & Identity Providers**. If you use the Classic UI, hover over **Security**, and then click **Identity Providers**.
```
Comment (larsjohansen-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-in-with-facebook/configure-idp/index.md]: Pt 2.2 (for other providers) is missing from Facebook. I don't think that is intentional

File: packages/@okta/vuepress-site/docs/guides/sign-in-with-microsoft/configure-idp/index.md (hunk 13 above)
Comment (bellatumaneng-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-in-with-microsoft/configure-idp/index.md]: Yes – I checked the docs for Microsoft and Facebook for how to set up the app on their side, and the docs actually refer these as the "App ID/Secret."

File: packages/@okta/vuepress-site/docs/guides/sign-in-with-facebook/configure-idp/index.md (hunk 14 above)
Comment (bellatumaneng-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-in-with-facebook/configure-idp/index.md]: Oh, that's a good point. LinkedIn is missing it too.

**Pull request merged**